CauldronMods/Controller/Heroes/Gargoyle/CharacterCards/GargoyleCharacterCardController.cs

```diff
@@ -10,33 +10,170 @@ namespace Cauldron.Gargoyle
+                increaseDamageStatusEffect = new IncreaseDamageStatusEffect(INCREASE_DAMAGE_POWER)
+                {
+                    SourceCriteria = { IsSpecificCard = base.CharacterCard },
+                    NumberOfUses = NUMBER_USES_ONCE
+                };
+                coroutine = base.AddStatusEffect(increaseDamageStatusEffect);
+                if (base.UseUnityCoroutines)
+                {
+                    yield return base.GameController.StartCoroutine(coroutine);
+                }
+                else
+                {
+                    base.GameController.ExhaustCoroutine(coroutine);
+                }
+            }
+
             yield break;
         }
 
+        /* 
+         * Incap
+         * "Increase the next damage dealt by a hero target by 2."
+         * "Reduce the next damage dealt to a hero target by 2."
+         * "One player may draw a card now."
+        */
         public override IEnumerator UseIncapacitatedAbility(int index)
         {
             IEnumerator coroutine;
+
             switch (index)
             {
+                // "Increase the next damage dealt by a hero target by 2."
                 case 0:
                     {
+                        coroutine = this.DoIncapacitateOption1();
+                        if (base.UseUnityCoroutines)
+                        {
+                            yield return base.GameController.StartCoroutine(coroutine);
+                        }
+                        else
+                        {
+                            base.GameController.ExhaustCoroutine(coroutine);
+                        }
                         break;
                     }
                 case 1:
+                    // "Reduce the next damage dealt to a hero target by 2."
                     {
+                        coroutine = this.DoIncapacitateOption2();
+                        if (base.UseUnityCoroutines)
+                        {
+                            yield return base.GameController.StartCoroutine(coroutine);
+                        }
+                        else
+                        {
+                            base.GameController.ExhaustCoroutine(coroutine);
+                        }
                         break;
                     }
                 case 2:
+                    // "One player may draw a card now."
                     {
+                        coroutine = this.DoIncapacitateOption3();
+                        if (base.UseUnityCoroutines)
+                        {
+                            yield return base.GameController.StartCoroutine(coroutine);
+                        }
+                        else
+                        {
+                            base.GameController.ExhaustCoroutine(coroutine);
+                        }
                         break;
                     }
             }
             yield break;
         }
+
```

Comment:
Style choice, but given how small the Incap's are I would not have the extra level of functions.  It's worth noting that the game engine only evaluates a single 'coroutine' loop each frame (in the worst case) so there is a cost to nesting coroutine calls needlessly.